scalereg/reg6/staff/views.py

```diff
@@ -181,6 +201,90 @@ def CashPayment(request):
     })
 
 
+@login_required
+def CashPaymentRegistered(request):
```

Comment:
This commit contains a lot of unrelated changes. We should separate them to make the code clearer. Should I upstream some of this code first?

Reply:
If you can that'd be great.

Reply:
Working on that. There are a few dependencies, so some PRs will have to be merged before others. Hopefully in a day or two.

Reply:
rebased on master, which should have this addressed now based on your other PRs.